app/controllers/v0/triage_teams_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module V0
+  class TriageTeamsController < HealthcareMessagingController
+    def index
+      @teams = client.get_triage_teams
```

Comment:
is it necessary to use instance variables? I don't think these variables are being used in any other methods so local variables should suffice.